tests/api_connexion/endpoints/test_dataset_endpoint.py

```diff
@@ -53,21 +53,23 @@ class TestDatasetEndpoint:
     default_time = "2020-06-11T18:00:00+00:00"
 
     @pytest.fixture(autouse=True)
-    def setup_attrs(self, configured_app) -> None:
+    def setup(self, configured_app):
```

Comment:
Don’t call this `setup`, the function name has special meaning to Pytest (it would not pick up that special meaning here due to the fixture decorator, but it’s better to not confuse things)